ofl/overpass/DESCRIPTION.en_us.html

```diff
@@ -1 +1,13 @@
-<p>An open source font family inspired by Highway Gothic. Sponsored by <a href="https://www.redhat.com/">Red Hat</a> &mdash; Created by <a href="https://www.delvefonts.com">Delve Fonts</a>.</p>
\ No newline at end of file
+<p>Overpass is a free, Open Source typeface designed by <a href="https://www.delvefonts.com">Delve Fonts</a> initially on commission for Red Hat, Inc. Development of Overpass v4.0 was made possible with sponsorship by Google Fonts and with donations by individuals from all around the world — Thank You!</p>
+<p>The design of Overpass is an interpretation of the well-known “Highway Gothic” letterforms from the Standard Alphabets for Traffic Control Devices published by the U.S. Federal Highway Administration. Starting from those specifications, critical adjustments were made to the letterforms to create an optimal presentation at smaller sizes on-screen and later for display sizes, especially in the lighter weights.</p>
+<p>Also see <a href="https://fonts.google.com/specimen/Overpass+Mono">Overpass Mono!</a></p>
+<p>Overpass can be used for everything from extended passages of text in print and online, to UI design, posters, wayfinding signage, and environmental graphic design.</p>
+
+<ul>
+<li>Designers: Delve Withrington, Dave Bailey, and Thomas Jockin</li>
```

Comment:
All these designers should me listed in metadata.pb

Reply:
OK!